src/index.ts

```diff
@@ -20,8 +20,7 @@ import {
   setLogLevel,
 } from "./utils";
 import { getEnhancedMetricTags } from "./metrics/enhanced-metrics";
-
-export { TraceHeaders } from "./trace";
+import { DatadogTraceHeaders } from "trace/context/extractor";
```

Comment:
also somehow we broke the types here
<img width="1252" alt="image" src="https://github.com/DataDog/datadog-lambda-js/assets/1598537/b17397fb-06d8-4bc9-b872-1a59a8375ab5">

Reply:
you have to provide relative path as `../../trace/my-file.ts`

Reply:
hehe yeah just a TODO note for myself